### PR TITLE
The task with channels

### PR DESCRIPTION
**Pull request by vantihovich**

### CODE REVIEW

File: channels/task/main.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	go CheckEven(a, alarmClock) //starting 1st goroutine that checks if received value is even
+
+	go func() { //goroutine for timer
+		time.Sleep(4 * time.Second) //timer
+		alarmClock <- true          // sending the value to the stopper channel
+		close(a)
+		defer close(alarmClock)
+	}()
+
+	for i := 1; ; i += 1 { //endless cycle
```
Comment (apaliavy) [inline channels/task/main.go:21]: you can do just for {}

File: channels/task/main.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+}
+
+func CheckEven(ch chan int, alarm chan bool) {
+	b := make(chan int)    //the second channel
+	go PrintEven(b, alarm) //starting 2nd goroutine for printing even numbers
+
+	for {
+		select { //listening to both channels(channel with value, stopper channel)
+		case <-alarm: // if alarm channel received value - close the 2nd channel and stop the routine
```
Comment (apaliavy) [inline channels/task/main.go:41]: can we move this check to the main function?

File: channels/task/main.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+				b <- value //sending the value to the 2nd channel
+			}
+		}
+	}
+}
+
+func PrintEven(ch chan int, alarm chan bool) {
+	for {
+		select { //listening to both channels(channel with value, stopper channel)
+		case <-alarm: // if alarm channel received value - stop the routine
```
Comment (apaliavy) [inline channels/task/main.go:55]: the same - just close it in the caller (main)

File: channels/task/main.go (hunk 1 above)
Comment (vantihovich) [reply channels/task/main.go]: done

File: channels/task/main.go (hunk 2 above)
Comment (vantihovich) [reply channels/task/main.go]: done

File: channels/task/main.go (hunk 3 above)
Comment (vantihovich) [reply channels/task/main.go]: done

**Review by apaliavy — APPROVED** [review]

**Pull request merged**